### PR TITLE
Ll heuristic car main

### PR DESCRIPTION
**Pull request by akmaralAW**
Minor changes related to: 
1. using new low-level focal heuristic allowing car trailer example
2. table 5 is adjusted to have db-ecbs.

This branch has been benchmarked for table 5.

### CODE REVIEW

File: CMakeLists.txt
Comment (whoenig) [inline CMakeLists.txt:1]: Why were changes in this file needed?

File: example/algorithms.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ db-cbs:
   default:
     delta_0: 0.5
     delta_rate: 0.9
-    num_primitives_0: 300
+    num_primitives_0: 100
```
Comment (whoenig) [inline example/algorithms.yaml:43]: any reason to change this over main?

**Review by whoenig — APPROVED** [review]

File: CMakeLists.txt
Comment (akmaralAW) [reply CMakeLists.txt:1]: old functions that were used for debugging.

File: example/algorithms.yaml (hunk 1 above)
Comment (akmaralAW) [reply example/algorithms.yaml:43]: for the fair benchmarking with db-cbs. At_goal in db-cbs works only if motions are 100.

**Comment by akmaralAW** [comment]
tdbastar_epsilon uses a map <robot_id, <robot objects for collision checking>> in order to support car_trailer.

**Pull request merged**